Automattic/wp-calypso
# Add new skip button style to design picker step

#### Changes proposed in this Pull Request

* Originally, the skip button shows at the bottom of the page. So, I add `skipButtonAlign` to make the skip button style match the screenshot in [this issue](https://github.com/Automattic/wp-calypso/issues/55783)

##### Deskop

![image](https://user-images.githubusercontent.com/13596067/131493225-27c60696-0f82-4e7d-9ed7-953ccd15853a.png)

##### Mobile

![image](https://user-images.githubusercontent.com/13596067/131655939-4573e766-90ae-4dae-9bb5-bf03056e490c.png)

#### Testing instructions



* Open `/start/setup-site?siteSlug=<your_site>`
* Check skip button



Related to https://github.com/Automattic/wp-calypso/issues/55783
